core/src/services/cloudService.ts

```diff
@@ -1,4 +1,25 @@
-import { StringParams } from "../common/stringParams";
+import {  StringParams } from "../common/stringParams";
```

Comment:
nit: extra space after the curly brace

Reply:
Thanks for this feedback, It will be apply in my next commit